2-multitenant/modules/env_baseline/firewall.tf

```diff
@@ -0,0 +1,53 @@
+resource "google_compute_firewall" "allow_internal_ingress" {
```

Comment:
Hey @caetano-colin - If you are looking to apply these to the cluster nodes, you will something like `network_tags = [allow_internal_ingress]` added to the cluster definitions below.

That said, I might suggest adding `add_cluster_firewall_rules = true` for now to the cluster definitions.

Reply:
outdated, seems like we don't actually need the firewall rules